snippets/properties.json

```diff
@@ -0,0 +1,13 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
+			"quarkus.datasource.driver=${2:org.mariadb.jdbc.Driver}",
```

Comment:
I don't know if it works, but if choice (LSP) is working we could provide a list of JDBC driver?